app/lib/import_export_constants.rb

```diff
@@ -522,7 +523,7 @@ def self.rct_node(schema, label, fields)
   HISTORIES_EXPORT_OPTIONS = {
     label: 'Histories',
     nodes: [rct_node(:histories, 'History', %i[patient_id user_defined_id_statelocal user_defined_id_cdc user_defined_id_nndss id created_by history_type
-                                               comment created_at updated_at])]
+                                               comment was_edited created_at updated_at])]
```

Comment:
This adds 'Edited' in custom export, but not full exports. Please also add this to the `FULL_HISTORY_HISTORIES_FIELDS` and `FULL_HISTORY_HISTORIES_HEADERS` (unless there was a deliberate decision I missed to not do that).

Reply:
Kelly just confirmed to add this to full history exports as well as the last field after created/updated at

Reply:
this has been removed for now as the fields will be changing, but will add back with the right fields once implemented

Reply:
This has been updated